Documentation/planning/arcade-powered-source-build/onboarding/README.md

```diff
@@ -5,16 +5,36 @@ Onboarding to ArPow (arcade-powered source-build) has several stages:
 * **(1) Local build infra merged**
    * See [local-onboarding.md](local-onboarding.md).
    * PR submitted by source-build team.
+   * Exit criteria:
```

Comment:
I don't think the exit criteria should be included in this README.  This will be a maintenance headache.  This is an overview and there's a link to the implemenation-plan below which contains the details.

Reply:
Removed the duplication.